addons/crm/models/crm_lead.py

```diff
@@ -1447,7 +1447,7 @@ def _merge_get_fields_specific(self):
         return {
             'description': lambda fname, leads: '<br/><br/>'.join(desc for desc in leads.mapped('description') if not is_html_empty(desc)),
             'type': lambda fname, leads: 'opportunity' if any(lead.type == 'opportunity' for lead in leads) else 'lead',
-            'priority': lambda fname, leads: max(leads.mapped('priority')) if leads else False,
+            'priority': lambda fname, leads: max(filter(None, leads.mapped('priority'))) if leads else False,
```

Comment:
`max([])` raises ValueError so I guess it should probably be `max(priorities) if (priorities := leads.filtered('priority').mapped('priority'))) else False` or something like that ideally?